### ISSUE TITLE
Missing Arcane Powder options

### ISSUE THREAD

**Issue by mjblack** [cliaz/Titan-Panel-Classic-Reagent-Tracker#16]
After the most recent update the reagent tracker is no longer showing arcane powder count, options for arcane powder, or buying arcane powder as previously configured. 

![image](https://user-images.githubusercontent.com/970638/195995201-fd5f557b-3b13-4c73-a4ed-bcf672c60eb4.png)
![image](https://user-images.githubusercontent.com/970638/195995257-be1d9ca2-f981-4ebc-82d9-4b15935e06a1.png)

**Comment by mjblack** [comment]
It looks like what is missing is Arcane Brilliance (Rank 3) (spell ID 43002) and Ritual of Refreshment (Rank 2) (spell ID 58659).

**Comment by cliaz** [comment]
Those spell additions look good, I’ll merge that PR shortly - thanks!

Thing is, them being missing shouldn’t cause the issue you’re describing. I assume you still have AB rank 1 and 2 and RoR, so the addon should pick them up and track the reagent.
Have you tested your changes, do they fix things?

**Comment by mjblack** [comment]
I'm not sure why they disappeared but I do have those spells. I'm not sure how it tracks spells, whether its just trained or if they're on an action bar. I did test out the change and I was able to see the reagent option show back up. After the change, the addon is working as expected.

![image](https://user-images.githubusercontent.com/970638/196036985-e5f69276-e9e5-40f7-b44a-ded20f44751c.png)
![image](https://user-images.githubusercontent.com/970638/196037201-04193e0e-b978-4fee-bffa-72434d62a759.png)
